slashing-protection/src/integration-test/java/tech/pegasys/web3signer/slashingprotection/InterchangeBaseIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.slashingprotection;
+
+import tech.pegasys.web3signer.slashingprotection.interchange.InterchangeModule;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.Resources;
+import com.opentable.db.postgres.embedded.EmbeddedPostgres;
+import org.flywaydb.core.Flyway;
+
+public class InterchangeBaseIntegrationTest {
+
+  protected final ObjectMapper mapper = new ObjectMapper().registerModule(new InterchangeModule());
+
+  protected EmbeddedPostgres setup() throws IOException, URISyntaxException {
+    final EmbeddedPostgres slashingDatabase = EmbeddedPostgres.start();
```

Comment:
just wondering, do we have to close the embedded postgresql as well?

Reply:
good point - closing it now.